### PR TITLE
[231052] Upgrade Erlang to 24 and Elixir to 1.13

### PR DESCRIPTION
**Pull request by aYukiSekiguchi**
https://acsmine.tok.access-company.com/redmine/issues/231052

### CODE REVIEW

**Review by Double-oxygeN — DISMISSED** [review]
LGTM

File: mix.exs
Comment (Double-oxygeN) [inline mix.exs:1]: The required Elixir version should be 1.13+ after this change. Could you change the requirement in `mix.exs` and also change `CHANGELOG.md`?

Comment (aYukiSekiguchi) [reply mix.exs:1]: Good catch!
Fixed at https://github.com/access-company/antikythera_aws/pull/25/commits/c58ae4810d81f468c1d452ecdafce0fc68c1705f

**Review by Double-oxygeN — APPROVED** [review]
LGTM

**Pull request merged**